test/distributions/robust/arithmetic/modules/test_module_Sort.chpl

```diff
@@ -3,10 +3,11 @@
 use driver_real_arrays;
 use Random;
 use Sort;
+use BubbleSort, MergeSort, SelectionSort, HeapSort;
 
 var rng = new randomStream(real, 314159265);
 
-enum SortType { BUBBLE=0, INSERTION, MERGE, SELECTION, QUICK, HEAP };
+enum SortType { BUBBLE=0, INSERTION, MERGE, SELECTION, QUICK, HEAP }
```

Comment:
Why no semicolon? :^(
Does our style guide say no semicolon for `enum`s?

Reply:
Semicolons on enums are not required in Chapel, and the parsed AST is two statements
1. the Enum
2. an EmptyStmt

I don't believe its in the style guide, but `chplcheck` does warn for it